packages/calling/src/CallHistory/CallHistory.ts

```diff
@@ -308,6 +369,15 @@ export class CallHistory extends Eventing<CallHistoryEventTypes> implements ICal
     }
   };
 
+  handleUserSessionsDeletedEvents = async (event?: CallSessionDeletedEvent) => {
```

Comment:
I am curious why event is optional here since this is a callback for an event - the event object should always be present.

Reply:
The event parameter is marked as optional (event?) due to TypeScript's type-checking, which expects the callback to handle both CallSessionDeletedEvent and undefined. Removing the optional marker causes a type error since it won’t match the expected signature. We also follow this pattern in other event handlers, such as CallSessionEvent and CallSessionViewedEvent, to maintain consistency.